.travis.yml

```
@@ -5,6 +5,7 @@ matrix:
   include:
     - go: 1.12.x
     - go: 1.13.x
+    - go: 1.14
```

It should be 1.14.x to introduce the latest 1.14.1

```suggestion
    - go: 1.14.x
```

---

Done! ;)